lib/clowne/adapters/sequel/associations/one_to_many.rb

```diff
@@ -7,12 +7,13 @@ module Associations
         class OneToMany < Base
           def call(record)
             clones =
-              with_scope.map do |child|
+              with_scope.lazy.map do |child|
```

Comment:
What is the reason for `lazy` here?

Reply:
I chain two `map` via `lazy` to avoid double iteration